zerver/models.py

```diff
@@ -555,7 +555,7 @@ class UserProfile(AbstractBaseUser, PermissionsMixin):
     MAX_NAME_LENGTH = 100
     MIN_NAME_LENGTH = 3
     API_KEY_LENGTH = 32
-    NAME_INVALID_CHARS = ['*', '`', '>', '"', '@']
+    NAME_INVALID_CHARS = ['*', '`', '>', '"', '@', '#']
```

Comment:
Do we need this change to be part of this commit?  It seems like we have two versions of NAME_INVALID_CHARS in `models.py`.

Reply:
Ok, ideally these would be in separate commits.

Reply:
Okay, I'll update this.